analyzers/src/SonarAnalyzer.CSharp/Rules/UninvokedEventDeclaration.cs

```diff
@@ -69,9 +69,12 @@ private void RaiseOnUninvokedEventDeclaration(SymbolAnalysisContext context)
                 return;
             }
 
+            var invokedSymbols = GetInvokedEventSymbols(removableDeclarationCollector);
+            var possiblyCopiedSymbols = GetPossiblyCopiedSymbols(removableDeclarationCollector);
+
             removableEventFields
-                .Where(IsNotInvoked)
-                .Where(IsNotCopied)
+                .Where(tuple => !invokedSymbols.Contains(tuple.Symbol))
+                .Where(tuple => !possiblyCopiedSymbols.Contains(tuple.Symbol))
```

Comment:
I'd go for `x` as it's pretty flat case and it's obvious that we're iterating `removableEventFields`. There's also `x` below in `ForEach`
```suggestion
                .Where(x => !invokedSymbols.Contains(x.Symbol))
                .Where(x => !possiblyCopiedSymbols.Contains(x.Symbol))
```